### ISSUE TITLE
Old school platformers and training set

### ISSUE THREAD

**Issue by Bloodmosher** [MineDojo/NitroGen#13]
Do you think it should be able to play old school platform games like super mario bros,  giana sisters, etc? 

I've hooked it up to this one but it is pretty terrible...

<img width="304" height="200" alt="Image" src="https://github.com/user-attachments/assets/992b4c0a-be6b-47cd-ad45-0557e83a3031" />

**Comment by loicmagne** [comment]
> Cool any pointers to how I can go about that? I have a good setup for collecting labeled frames.

You can start from here to see how we compute loss https://github.com/MineDojo/NitroGen/blob/main/nitrogen/flow_matching_transformer/nitrogen.py#L497-L565  

The full fine-tuning code isn't released yet
